Dockerfile

```diff
@@ -177,6 +177,12 @@ COPY --chown=${USER} ssh ${HOME}/.ssh
 COPY --chown=${USER} wait-for-it.sh manage.py backend_entrypoint.sh ${HOME}/
 COPY --chown=${USER} utils/ ${HOME}/utils
 COPY --chown=${USER} cvat/ ${HOME}/cvat
+COPY --chown=${USER} tests/python/.coveragerc ${HOME}/
```

Comment:
Can we eliminate having .coveragerc file in production docker image? For example maybe we can mount this file in docker-compose.dev.yml or in `create_compose_files` in fixtures/init.py

Reply:
Updated